src/Helper/HeadScript.php

```diff
@@ -33,6 +33,7 @@ class HeadScript extends Placeholder\Container\AbstractStandalone
      */
     const FILE   = 'FILE';
     const SCRIPT = 'SCRIPT';
+    const DEFAULT_SCRIPT_TYPE = 'text/javascript';
```

Comment:
This constant should also be used in `Laminas\View\Helper\InlineScript`:

https://github.com/laminas/laminas-view/blob/9512a1939a67cec3deb8e841b651505424f1e168/src/Helper/InlineScript.php#L30-L36

Reply:
Like `$type = HeadScript::DEFAULT_SCRIPT_TYPE`
or by making a separated "Enum" Class?

Reply:
The `InlineScript` helper is an extension of the `HeadScript` helper:

https://github.com/laminas/laminas-view/blob/9512a1939a67cec3deb8e841b651505424f1e168/src/Helper/InlineScript.php#L15

Therefore: `$type = self::DEFAULT_SCRIPT_TYPE`

Reply:
Oh, I've overlooked this fact.

Reply:
Probably, as we cannot add visibility on the constant, we should add php annotation `@internal` for now. Then in next minor, when we bump php version we can use private/protected visibility.

Reply:
added your suggestion @michalbundyra